src/Queue/LaravelSqsHandler.php

```diff
@@ -89,6 +92,10 @@ private function process(string $connectionName, SqsJob $job): void
 
             $this->raiseAfterJobEvent($connectionName, $job);
         } catch (Throwable $e) {
+            // Report exception to defined log channel
+            $this->exceptions->report($e);
+
+            // Fire off an exception event
             $this->raiseExceptionOccurredJobEvent($connectionName, $job, $e);
```

Comment:
That looks awesome, thanks for the changes.

One minor thing I would do is switch those lines to preserve the order in the Laravel code base:

```suggestion
            // Fire off an exception event
            $this->raiseExceptionOccurredJobEvent($connectionName, $job, $e);

            // Report exception to defined log channel
            $this->exceptions->report($e);
```